beacon-chain/core/epoch/participation.go

```diff
@@ -1,17 +1,28 @@
 package epoch
 
 import (
+	"fmt"
+
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
 	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
 	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
 )
 
-// ComputeValidatorParticipation by matching validator attestations during the epoch,
-// computing the attesting balance, and how much attested compared to the total balances.
-func ComputeValidatorParticipation(state *pb.BeaconState) (*ethpb.ValidatorParticipation, error) {
-	currentEpoch := helpers.SlotToEpoch(state.Slot)
-	atts, err := MatchAttestations(state, currentEpoch)
+// ComputeValidatorParticipation by matching validator attestations from the previous epoch,
+// computing the attesting balance, and how much attested compared to the total balance.
```

Comment:
For peeps without proper context, let's add a comment to say why previous epoch is preferred than current epoch.